File: 8_Kyu_Kata/sum_of_positive/Python/sum_of_positive.py
# 8 kyu - Sum of positive  [ ID: 5715eaedb436cf5606000381  (sum-of-positive) ]
# URL: https://www.codewars.com/kata/5715eaedb436cf5606000381
# Category: NONE  |  Tags: ARRAYS | FUNDAMENTALS
# ******************************************************************************
def positive_sum(arr):
    def positive_check(value):
        return value >= 0

    return sum(filter(positive_check, arr))

File: 8_Kyu_Kata/sum_of_positive/JavaScript/sumOfPositive.js
// 8 kyu - Sum of positive  [ ID: 5715eaedb436cf5606000381  (sum-of-positive) ]
// URL: https://www.codewars.com/kata/5715eaedb436cf5606000381
// Category: undefined  |  Tags: ARRAYS | FUNDAMENTALS
// *****************************************************************************
const positiveSum = (arr) => (!!arr ? arr.filter((v) => v >= 0).reduce((a, b) => a + b, 0) : 0)
